decidim-elections/db/migrate/20220424121541_add_hide_can_i_vote_to_votings.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddHideCanIVoteToVotings < ActiveRecord::Migration[6.1]
+  def change
+    add_column :decidim_votings_votings, :hide_can_i_vote, :boolean, default: false
```

Comment:
```suggestion
    add_column :decidim_votings_votings, :show_check_census, :boolean, default: false
```
(Mind that it also need to be renamed the filename)